test/unit/entity/acl.test.js

```diff
@@ -0,0 +1,319 @@
+import { Acl } from 'src/entity';
+import { randomString } from 'src/utils';
+import expect from 'expect';
+
+describe('Acl', function() {
```

Comment:
For `ACL` and `Metadata`, the tests you've added are good. 
One test I suggest adding for each (if it doesn't already exist) is to mock an entity response that has `acl` and `kmd` values, and make sure they're deserialized to the right objects. This will ensure, for example, that we're reading the correct JSON keys for readers / writers / dates etc.